packages/clerk-js/src/ui/Components.tsx

```diff
@@ -518,9 +518,13 @@ const Components = (props: ComponentsProps) => {
         )}
 
         {__BUILD_FLAG_KEYLESS_UI__
-          ? state.options?.__internal_claimKeylessApplicationUrl && (
+          ? state.options?.__internal_claimKeylessApplicationUrl &&
+            state.options?.__internal_claimKeylessApplicationUrl && (
```

Comment:
Is this the same thing twice?
```suggestion
          ? state.options?.__internal_claimKeylessApplicationUrl && (
```

Reply:
nice catch